src/main/java/de/retest/recheck/review/GlobalIgnoreApplier.java

```diff
@@ -32,59 +32,59 @@ public static GlobalIgnoreApplier create( final Counter counter, final Persistab
 	}
 
 	@Override
-	public boolean shouldIgnoreAttributeDifference( final Element element, final AttributeDifference difference ) {
-		return any( ignore -> ignore.shouldIgnoreAttributeDifference( element, difference ) );
+	public boolean shouldBeFiltered( final Element element, final AttributeDifference difference ) {
+		return any( attribute -> attribute.shouldBeFiltered( element, difference ) );
```

Comment:
Should be `filter` instead of `attribute`.

Reply:
I assumed it would fit better